src/patternfly/components/DataList/data-list.scss

```diff
@@ -333,7 +332,7 @@
 
   // offset action button
   .#{$data-list}__action {
-    margin-block-start: var(--#{$data-list}__action--MarginBlockStart);
+    margin-block-start: var(--#{$data-list}__item-action__action--MarginBlockEnd);
```

Comment:
* Replaces `__action` var with `__item-action`, was flagged to update in a breaking change in this PR/comment - https://github.com/patternfly/patternfly/pull/3750#pullrequestreview-552846443

Reply:
Should this be `--MarginBlockStart`?

Reply:
Oops! Updated and updated a `> *:not(:last-child)` inline margin to gap - https://github.com/patternfly/patternfly/pull/6851/commits/66af31ded741216ca9e1285eec7d8672206f7c81